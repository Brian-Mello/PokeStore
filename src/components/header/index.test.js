import React from 'react';
import renderer from 'react-test-renderer';
import Header from '.';

describe('testando integridade do Header component', () => {
    it(' Header snapshot', () => {
        const header = renderer.create(<Header/>).toJSON();
        expect(header).toMatchSnapshot();
    });
}); 