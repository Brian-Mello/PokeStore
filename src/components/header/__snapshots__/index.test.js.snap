// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testando integridade do Header component  Header snapshot 1`] = `
<header
  className="sc-AxjAm fWTOyA"
>
  <div
    className="sc-AxirZ bKoaym"
  >
    <div
      className="sc-AxgMl jVckvC"
    >
      <h1
        className="sc-Axmtr cHtizg"
      />
    </div>
  </div>
  <div
    className="sc-AxiKw fvGISm"
  >
    <div
      className="MuiFormControl-root MuiTextField-root sc-AxhUy gFTGAK"
    >
      <div
        className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl MuiInputBase-adornedStart"
        onClick={[Function]}
      >
        <div
          className="MuiInputAdornment-root MuiInputAdornment-positionStart"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
            />
          </svg>
        </div>
        <input
          aria-invalid={false}
          autoFocus={false}
          className="MuiInputBase-input MuiInput-input MuiInputBase-inputAdornedStart"
          disabled={false}
          id="outlined-basic"
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={false}
          type="text"
        />
      </div>
    </div>
  </div>
  <div
    className="sc-AxhCb hOmrwj"
  >
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-contained sc-AxmLO kYEaXQ"
      disabled={false}
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M10 6L8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
          />
        </svg>
      </span>
    </button>
  </div>
</header>
`;
